packages/playground/src/weblets/profile_manager.vue

```diff
@@ -193,7 +190,11 @@
                           color="secondary"
                           variant="outlined"
                           :disabled="
-                            isValidForm || !!mnemonic || shouldActivateAccount || keypairType === KeypairType.ed25519
+                            (isValidForm ||
+                              !!mnemonic ||
+                              shouldActivateAccount ||
+                              keypairType === KeypairType.ed25519) &&
+                            !shouldCreateAccount
```

Comment:
if the user entered a valid mnemonic, and doesn't have twin, the expected behavior is to click on connect then and it will create an account with the provided mnemonic.


[Screencast from 2024-09-01 11-31-34.webm](https://github.com/user-attachments/assets/9cf55029-7c08-4855-a15a-134e19417472)


on dashboard 2.5.0
[Screencast from 2024-09-01 11-32-32.webm](https://github.com/user-attachments/assets/b0560e5e-6fe5-449b-8c6d-2c1945530161)